js/modules/k6/http/http_request.go

```diff
@@ -265,6 +266,9 @@ func (h *HTTP) request(ctx context.Context, rt *goja.Runtime, state *common.Stat
 		if res.Header.Get("Content-Encoding") == "deflate" {
 			res.Body, resErr = zlib.NewReader(res.Body)
 		}
+		if res.Header.Get("Content-Encoding") == "gzip" {
```

Comment:
Please make this an `else if`.

Reply:
Thanks for your reply. A "switch" statement could it be better ?

Reply:
Yup, a switch statement sounds good.